### PR TITLE
[chess game] section styling

### LINKED ISSUE #4502: [chess game] section styling
**Describe the Feature you want to end**
I would like to enhance section design

**Expected behavior**
it helps in increasing the user interaction to the site

**Screenshots (optional)**

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by SnigdhaP2022**
# Chess game Background styling

Close #4502 

# Checklist:

- [✅] I have mentioned the issue number in my Pull Request.
- [✅] I have commented my code, particularly in hard-to-understand areas

# Before
Screenshot 2023-08-03 at 5 03 11 PM

# After
Screenshot 2023-08-03 at 5 03 29 PM

### CODE REVIEW

**Comment by SnigdhaP2022** [comment]
@TusharKesarwani  Kindly,review this

**Comment by SnigdhaP2022** [comment]
@MohitGupta121 @TusharKesarwani Kindly, review this

File: Frontend-Projects/Chess Website/chess.jpeg
Comment (MohitGupta121) [inline Frontend-Projects/Chess Website/chess.jpeg:1]: @SnigdhaP2022 remove this image from PR and use it as a link after uploading somewhere else.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: Frontend-Projects/Chess Website/chess.jpeg
Comment (SnigdhaP2022) [reply Frontend-Projects/Chess Website/chess.jpeg:1]: @MohitGupta121 Issue resolved. Kindly, review and merge this PR

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**